src/dev/build/tasks/bin/scripts/kibana.bat

```diff
@@ -14,7 +14,25 @@ If Not Exist "%NODE%" (
   Exit /B 1
 )
 
-set "NODE_OPTIONS=--no-warnings --max-http-header-size=65536 %NODE_OPTIONS%" && "%NODE%" "%DIR%\src\cli" %*
+set CONFIG_DIR=%KIBANA_PATH_CONF%
+If [%KIBANA_PATH_CONF%] == [] (
+  set CONFIG_DIR=%DIR%config
```

Comment:
We need a path separator here:
```suggestion
  set CONFIG_DIR=%DIR%\config
```

Additionally, can we protect against `node.options` not being available. It's logging an error otherwise which isn't in our format that is especially an issue when logging JSON.

`The system cannot find the file C:\Users\Tyler\Desktop\config\node.options.`

Reply:
Solved the `The system cannot find the file C:\Users\Tyler\Desktop\config\node.options.` error message